modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
-// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo    
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo
+// * modules/openshift-developer-cli-creating-and-pushing-a-component-from-client-os-to-the-restricted-cluster
```

Comment:
Again, shouldn't be including a module from another module.

Reply:
Restructured